services/indexer/indexer.go

```diff
@@ -8,7 +8,7 @@ import (
 	pg "github.com/go-pg/pg/v10"
 	cid "github.com/ipfs/go-cid"
 	logging "github.com/ipfs/go-log/v2"
-	trace "go.opentelemetry.io/otel/api/trace"
+	"github.com/opentracing/opentracing-go"
```

Comment:
The ORM we are using supports tracing out-of-the-box: https://pg.uptrace.dev/tracing/. Are there any advantages to using opencensus over opentelemetry?